applications/projects/working-with-projects.adoc

```diff
@@ -37,26 +37,53 @@ In {product-title} clusters that use the Customer Cloud Subscription (CCS) model
 ====
 endif::openshift-dedicated[]
 
-include::modules/creating-a-project-using-the-web-console.adoc[leveloffset=+1]
+[id="working-with-projects-create-project"]
+== Creating a project
 
-include::modules/odc-creating-projects-using-developer-perspective.adoc[leveloffset=+1]
+You can use the {product-title} web console or the OpenShift CLI (oc) to create a project in your cluster.
 
-include::modules/creating-a-project-using-the-CLI.adoc[leveloffset=+1]
+include::modules/creating-a-project-using-the-web-console.adoc[leveloffset=+2]
 
-include::modules/viewing-a-project-using-the-web-console.adoc[leveloffset=+1]
+// include modules/odc-creating-projects-using-developer-perspective.adoc[leveloffset=+2]
 
-include::modules/viewing-a-project-using-the-CLI.adoc[leveloffset=+1]
+.Additional resources
+
+* xref:../../applications/projects/working-with-projects.adoc#odc-customizing-available-cluster-roles-using-the-web-console_projects[Customizing the available cluster roles using the web console]
+
+include::modules/creating-a-project-using-the-CLI.adoc[leveloffset=+2]
+
+[id="working-with-projects-viewing-project"]
+== Viewing a project
+
+You can use the {product-title} web console or the OpenShift CLI (oc) to view a project in your cluster.
+
+include::modules/viewing-a-project-using-the-web-console.adoc[leveloffset=+2]
+
+include::modules/viewing-a-project-using-the-CLI.adoc[leveloffset=+2]
 
 include::modules/odc-providing-project-permissions-using-developer-perspective.adoc[leveloffset=+1]
 
-include::modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc[leveloffset=+1]
+include::modules/odc-customizing-available-cluster-roles-using-the-web-console.adoc[leveloffset=+1]
 
 include::modules/adding-to-a-project.adoc[leveloffset=+1]
 
-include::modules/checking-project-status-using-the-web-console.adoc[leveloffset=+1]
+[id="working-with-projects-viewing-project-status"]
+== Checking the project status
+
+You can use the {product-title} web console or the OpenShift CLI (oc) to view the status of your project.
```

Comment:
`oc` is not properly formatted as monospace here. `{oc-first}` can be used instead.